clang/test/Analysis/out-of-bounds-diagnostics.c

```diff
@@ -245,10 +265,15 @@ int *nothingIsCertain(int x, int y) {
   if (x >= 2)
     return 0;
   int *mem = (int*)malloc(x);
+
+  // TODO: The next line is a temporary hack; see 'symbolicExtent()' for details.
+  (void)x;
+
```

Comment:
These hacks are sort of well known. Just put these at the very end of the test-case. That's usually the canonical form of this hack. A comment like `Keep constraints alive.` is fine. Same applies for similar "hacks" in the tests of this PR.

Reply:
It's sad to hear that we need "sort of well known" hacks for this kind of issue.